data/events.toml

```diff
@@ -0,0 +1,56 @@
+[[events]]
+type = ["workshop"]
```

Comment:
I assume you already looked into it, but I presume there is no way for us to treat the "collection" of event posts as data? It just seems unfortunate we need to duplicate the metadata here as well as in the `/events/xxx.md` file. But I'll totally believe you if you say that isn't possible and/or prohibitively a PITA.

Reply:
... I see `tools/eventgen/main.go ` later in the PR, which makes this much more palatable. Though if it is valid, putting a header like the following would be much appreciated:

```
# DO NOT MODIFY
# This file is automatically generated by the ./tools/xxx. It is built from
# ...
```

Reply:
👍